packages/RNTester/js/utils/AsyncStorage/RCTAsyncStorage.expo.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Expo managed apps don't include the @react-native-community/async-storage
```

Comment:
@chirag-singhal , do we need this file?
It is for the expo apps.

Reply:
@stealthanthrax yeah we don't require that. Removing that!